examples/src/main/java/tachyon/examples/BasicRawTableOperations.java

```diff
@@ -49,10 +50,10 @@
   private long mId;
 
   public BasicRawTableOperations(TachyonURI masterAddress, TachyonURI tablePath,
-      WriteType writeType) {
+      TachyonStorageType writeType) {
     mMasterAddress = masterAddress;
     mTablePath = tablePath;
-    mWriteType = writeType;
+    mWriteType = writeType.isStore() ? WriteType.MUST_CACHE : WriteType.THROUGH;
```

Comment:
Right, I think we should take both under storage and tachyon write type. That will be the next step for refactoring Tachyon examples.